### PR TITLE
[16.0][ADD] loyalty_initial_date_validity: New module

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT45685

@chienandalu @ernesto-garcia-tecnativa please review

### CODE REVIEW

File: loyalty_initial_date_validity/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+To use this module simply go to edit or create a promotion and set a start date in
+the field labelled "Start date".
```
Comment (chienandalu) [inline loyalty_initial_date_validity/readme/USAGE.rst:2]: Well, point to the sale_... extension as the field here is useless on its own

**Review by chienandalu — APPROVED** [review]
LGTM :+1: 

Just a nitpick:

**Review by ernesto-garcia-tecnativa — APPROVED** [review]
LGTM

File: loyalty_initial_date_validity/readme/USAGE.rst (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply loyalty_initial_date_validity/readme/USAGE.rst:2]: done!

**Pull request merged**